src/components/sort-by-genre/SortByGenre.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './SortByGenre.css';
+
+class SortByGenre extends Component {
```

Comment:
It would be much more useful to give your components more saying names. Something like that: DetailsBar or SortBar and reuse it as possible